sklearn/utils/tests/test_optimize.py

```diff
@@ -28,7 +28,7 @@ def grad_hess(x):
         return grad(x), lambda x: A.T.dot(A.dot(x))
 
     assert_array_almost_equal(
-        _newton_cg(grad_hess, func, grad, x0, tol=1e-10)[0],
+        _newton_cg(grad_hess, func, grad, x0, tol=1e-7)[0],
```

Comment:
With a tolerance of 1e-10 I had two test failures. I was able to resolve these by reducing the threshold to 1e-7. It's not an optimal solution but the default of `tol` is 1e-4. So I think this solution should be acceptable.